src/Head2Head.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import {
+	getRaces,
+	getSlugData,
+	numPagesForGame,
+	RACETIME,
+	SlugData,
+} from "./Loader";
+import FormattedTable from "./FmtTable";
+import CategoryLauncher from "./CategoryLauncher";
+import "./Head2Head.css";
+import MUTable, { MUEntrant, MUMapCtor } from "./MUTable";
+
+// Used to cancel the async function that loads races if a new slug is set.
+let token: Symbol;
```

Comment:
This is a bit of an idiosyncratic way to handle cancellation. An alternative way to address this would be treating the loading task as state and storing a "cancel" callback there.